bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/PageImpl.java

```diff
@@ -56,6 +56,8 @@
     public static final String PN_REDIRECT_TARGET = "cq:redirectTarget";
 
     private Boolean hasCloudconfigSupport;
+    protected String[] headClientLibCategories = new String[0];
+    protected static final String PN_HEADCLIENTLIBS = "headclientlibs";
```

Comment:
• Perhaps this property could go to the Page model
• can we camelCase it "headClientlibs"